src/lib/Tests/Pagination/Pagerfanta/URLWildcardAdapterTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\Pagination\Pagerfanta;
+
+use eZ\Publish\API\Repository\URLWildcardService;
+use eZ\Publish\API\Repository\Values\Content\UrlWildcard;
+use Ibexa\AdminUi\Pagination\Pagerfanta\URLWildcardAdapter;
+use PHPUnit\Framework\TestCase;
+
+class URLWildcardAdapterTest extends TestCase
+{
+    /** @var \eZ\Publish\API\Repository\Values\Content\UrlWildcard|\PHPUnit\Framework\MockObject\MockObject */
+    private $urlWildcardService;
+
+    protected function setUp(): void
+    {
+        $this->urlWildcardService = $this->createMock(URLWildcardService::class);
+    }
+
+    public function testGetNbResults()
+    {
+        $countAll = 5;
+
+        $this->urlWildcardService
+            ->expects($this->once())
+            ->method('countAll')
+            ->with()
```

Comment:
Empty `with`, which in practice is an assertion definition, can & should be skipped.
```suggestion
```